controller/src/main/java/io/pravega/controller/server/SessionExpirationException.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.server;
+
+/**
+ * Base class for exceptions thrown from controller server code base.
```

Comment:
Is this comment correct? Seems a bit too broad given the exception at hand, which is specific to session expiration.